omnigibson/prims/rigid_prim.py

```diff
@@ -796,6 +798,51 @@ def visual_aabb_center(self):
         min_corner, max_corner = self.visual_aabb
         return (max_corner + min_corner) / 2.0
 
+    @cached_property
+    def is_meta_link(self):
+        # Check using the new format first
+        new_format = self.prim.HasAttribute("ig:isMetaLink") and self.get_attribute("ig:isMetaLink")
+        if new_format:
+            return True
+
+        # Check using the old format.
+        # TODO: Remove this after the next dataset release
+        old_format = m.LEGACY_META_LINK_PATTERN.fullmatch(self.name) is not None
+        if old_format:
+            return True
+
+        return False
+
+    @cached_property
+    def meta_link_type(self):
+        assert self.is_meta_link, f"{self.name} is not a meta link"
+        if self.prim.HasAttribute("ig:metaLinkType"):
+            return self.get_attribute("ig:metaLinkType")
+
+        # Check using the old format.
+        # TODO: Remove this after the next dataset release
+        return m.LEGACY_META_LINK_PATTERN.fullmatch(self.name).group(1)
+
+    @cached_property
+    def meta_link_id(self):
```

Comment:
Can we add some docstring about meta_link_id and sub_id? I actually don't quite remember what these are..